packages/app-project/src/screens/ProjectHomePage/components/ZooniverseTalk/components/JoinInButton/JoinInButton.js

```diff
@@ -1,24 +1,27 @@
 import { Button } from 'grommet'
-import Link from 'next/link'
-import { object } from 'prop-types'
 import styled from 'styled-components'
 import { useTranslation } from 'next-i18next'
+import { useRouter } from 'next/router'
+
+import addQueryParams from '@helpers/addQueryParams'
 
 const StyledButton = styled(Button)`
   text-align: center;
 `
 
-function JoinInButton (props) {
+function JoinInButton () {
   const { t } = useTranslation('screens')
-  const { linkProps } = props
+  const router = useRouter()
+  const { owner, project } = router?.query
   const label = t('Home.ZooniverseTalk.button')
+
+  // Talk is a PFE page
+  const href = addQueryParams(`https://www.zooniverse.org/projects/${owner}/${project}/talk`)
```

Comment:
I assume this is now a full URL instead of the older relative `/${owner}/${project}/talk` due to FEM-to-PFE routing shenanigans.

Reply:
Correct! I think this could still technically be `/${owner}/${project}/talk`, but I'm making it a full URL to be clear it's a link to an external app.